eventlogger/src/test/java/gov/cms/ab2d/eventlogger/eventloggers/kinesis/KinesisEventLoggerTest.java

```diff
@@ -0,0 +1,74 @@
+package gov.cms.ab2d.eventlogger.eventloggers.kinesis;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jdk8.Jdk8Module;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import com.fasterxml.jackson.module.paramnames.ParameterNamesModule;
+import gov.cms.ab2d.eventlogger.AB2DPostgresqlContainer;
+import gov.cms.ab2d.eventlogger.SpringBootApp;
+import gov.cms.ab2d.eventlogger.events.BeneficiarySearchEvent;
+import gov.cms.ab2d.eventlogger.events.ErrorEvent;
+import org.json.JSONException;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.testcontainers.containers.PostgreSQLContainer;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+
+import java.time.OffsetDateTime;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+@SpringBootTest(classes = SpringBootApp.class)
+@Testcontainers
+class KinesisEventLoggerTest {
+    @Container
+    private static final PostgreSQLContainer postgreSQLContainer = new AB2DPostgresqlContainer();
+
+    @Autowired
+    private KinesisEventLogger logger;
+
+    /*
+    @Test
```

Comment:
It actually does an actual sample call to AWS. I didn't want it in the unit test because it relies on outside systems, but it's not terrible to have an example.